<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SellingModeSessionWidget</class>
 <widget class="QWidget" name="SellingModeSessionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>231</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>State</string>
    </attribute>
    <widget class="QLabel" name="SellersListTitleLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Buyers:</string>
     </property>
    </widget>
    <widget class="QTreeView" name="buyersTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>521</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Terms</string>
    </attribute>
    <widget class="QLabel" name="minContractFeePrKbValueLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;fee&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="minRefundDelayValueLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;delay&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="minPriceTitleLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Minimum price/piece (sats):</string>
     </property>
    </widget>
    <widget class="QLabel" name="minRefundDelayTitleLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Minimum refund delay (s):</string>
     </property>
    </widget>
    <widget class="QLabel" name="maxNumberOfSellersValueLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;#sellers&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="minContractFeePrKbTitleLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Minimum contract fee (sats/Kb):</string>
     </property>
    </widget>
    <widget class="QLabel" name="maxNumberOfSellersTitleLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Maximum number of co-sellers:</string>
     </property>
    </widget>
    <widget class="QLabel" name="minPriceValueLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;price&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="changeTermsPushButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>130</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Change</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Policy</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
